app.css

```diff
@@ -0,0 +1,319 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: sans-serif;
+}
+*::before, *::after {
+  box-sizing: inherit;
+}
+.total {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+.header {
+  background-image: url(http://cdn.grindtv.com/wp-content/uploads/2015/08/image1-1.png);
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  z-index: -1;
+}
+}
+.header:before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: rgba(63, 76, 167, 0.7);
+}
+.header_navbar {
+  width: 75%;
```

Comment:
Set a `max-width` as a `rem` or `px` value to make this more predictable and so that things don't get too big on large screens.